tensorflow/tools/api/golden/v1/tensorflow.pbtxt

```diff
@@ -1900,6 +1900,14 @@ tf_module {
     name: "rint"
     argspec: "args=[\'x\', \'name\'], varargs=None, keywords=None, defaults=[\'None\'], "
   }
+  member_method {
+    name: "roi_align"
+    argspec: "args=[\'input\', \'rois\', \'spatial_scale\', \'pooled_height\', \'pooled_width\', \'sampling_ratio\', \'min_level\', \'max_level\', \'canonical_scale\', \'canonical_level\', \'debug\', \'name\'], varargs=None, keywords=None, defaults=[\'1\', \'1\', \'1\', \'-1\', \'2\', \'5\', \'224\', \'4\', \'False\', \'None\'], "
+  }
+  member_method {
+    name: "roi_align_grad"
```

Comment:
This should be hidden in an apidef file

Reply:
I  ran `bazel-bin/tensorflow/tools/api/tests/api_compatibility_test --update_goldens True` as suggested with the failed test. What is the trick to hide this from the command above?

Reply:
Make a file like https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/api_def/python_api/api_def_Cholesky.pbtxt with the right endpoint in it

Reply:
I have the file generated by `update_api_def` but you are saying that I need to edit it manually above. I am a bit scared to do manual edits to protobuf files. Is there a better way to create them? If not is there a way to validate the edited file?

Reply:
There are tests which will fail if the file is incorrectly generated.

You can also build tensorflow and look at the generated gen_*_ops.py files for the comments to see that they are well-formed.

Reply:
Should be done now